Cargo.toml

```diff
@@ -509,6 +510,13 @@ rlimit = "0.10.1"
 rand_pcg = "0.3.1"
 xattr = { workspace = true }
 
+
+[target.'cfg(all(target_family= "unix",not(target_os = "macos")))'.dev-dependencies]
+serde = { version = "1.0.202", features = ["derive"] }
```

Comment:
could you please document where they are used ?
and what for

Reply:
Yea, sure.